src/types.ts

```diff
@@ -153,6 +153,31 @@ export interface SamlOptions extends Partial<SamlSigningOptions>, MandatorySamlO
   disableRequestAcsUrl: boolean;
   samlAuthnRequestExtensions?: Record<string, unknown>;
   samlLogoutRequestExtensions?: Record<string, unknown>;
+  metadataExtensions?: {
```

Comment:
This is a misleading name. These aren't metadata extensions, they are metadata elements (line 443). I think this should be broken up into individual configuration values, like `metadataContactPerson` and `metadataOrganization`. Otherwise, I think this looks good.

I take it that for your use, you only need `ContactPerson` and `Organization`? That is totally fine with me; only implement the part of the spec you need. As long as they are spec compliant and work for you we're good :)

Reply:
Done